app/controllers/phrase_controller.rb

```diff
@@ -20,6 +19,9 @@ def pages_visited
   end
 
   def mentions
-    SurveyAnswer.find_by_sql(['select date(s.started_at) as date, count(m.id) as total_mentions from survey_phrases m join phrases p on p.id = m.phrase_id join survey_answers sa on sa.id = m.survey_answer_id join surveys s on s.id = sa.survey_id where p.id = ? group by(date(s.started_at)) limit 10;', @phrase.id])
+    SurveyPhrase
+      .mentions_by_date_range_for_phrase(@phrase, Date.new(2020, 4, 1), Date.new(2020, 4, 7))
+      .sort_by{ |date, _| date }
```

Comment:
I'd be tempted to make a temporary helper method you can remove later:

`SurveyPhrase.sorted_mentions_by_initial_test_data_dates`

...rather than repeat the dates in two places and risk them going out of sync.

Reply:
I thought about adding this into `SurveyPhrase` but I feel the model shouldn't be responsible for sorting data. In terms of dates, these may change between the pages - currently they're placeholders but we plan to add in date filters next.